KicadModTree/nodes/Footprint.py

```diff
@@ -73,5 +73,5 @@ def setPasteMarginRatio(self, value):
         # paste_margin_ratio is unitless between 0 and 1
         # GUI uses percentage so account for that input
         if value > 1:
-            value = value / 100.0
+            value = abs(value) / 100.0
```

Comment:
Calling `abs` is redundant here: the line is only run if value > 1 (and is therefore positive), so it's a no-op.